en/asset-workflow/prefab.md

```diff
@@ -1,79 +1,91 @@
 
-Optimization Policy can optimizes the instantiation time for the selected prefab, which is the time required to execute `cc.instantiate`. The values that can be set are:
+### Associating nodes to prefabs
 
- - **Auto adjust** (default)
+Select both a node in the scene and a prefab in the **Assets** panel, then click **Node -> Connect Node to Prefab** in the top menu bar to associate the selected node and the prefab.
+
+## Prefab Options
 
-   When set to this option, the engine automatically adjusts the optimization policy based on the number of instantiations. When you first create an instance, the behavior is the same as **For single instance creation**. **For multiple instance creation** will be automatically used after multiple creation.
+When selecting any prefab in the **Assets** panel, set its **Optimization Policy**, **Async Load Assets** and **Readonly** in the **Properties** panel, and click the **Apply** button on the top right after setting to take effect.
 
- - **For single instance creation**
+### Setting Optimization Policy
 
-   This option skips code generation for this prefab.
+The optimization policy optimizes the instantiation time of the selected prefab, i.e. the time required to execute `cc.instantiate`. Possible settings include **Auto Adjust** (default), **For Single Instance Creation**, and **For Multiple Instance Creation**.
 
- - **For multiple instance creation**
+- If set to **Auto Adjust**, the engine will automatically adjust the optimization policy based on the number of creations. When creating a prefab instance for the first time, the policy is equivalent to **For Single Instance Creation**, and will automatically switch to **For Multiple Instance Creation** after multiple creations.
 
-   This option enables code generation for this prefab.
+- If this prefab requires repeated execution of `cc.instantiate`, please select **For Multiple Instance Creation**, otherwise just keep the default **Auto Adjust**.
 
-If this prefab requires repeated execution of `cc.instantiate`, select **For multiple instance creation**, otherwise leave as the default **Auto adjust**.
+### Setting delayed asset loading
 
-> In the older version of the engine, the optimization was fixed to **For multiple instance creation** and it is great for situations where you need to create objects repeatedly. However, many people use prefab as a tool for multi-people collaboration or step-by-step loading. Basically, these prefabs are only instantiated once, resulting in slower node creation. The **Auto adjust** in the new version is a good solution to this problem.
+This item is disabled by default. If this item is enabled, it will delay the loading of other assets that the prefab depends on, improving the loading speed of some page-based games. For details, please refer to the [Enable Async Load Assets](scene-managing.md#enable-async-load-assets) documentation.
```

Comment:
```suggestion
This option is disabled by default. If this option is enabled, it will delay the loading of other assets that the prefab depends on, improving the loading speed of some page-based games. For details, please refer to the [Enable Async Load Assets](scene-managing.md#enable-async-load-assets) documentation.
```